BlueDBOnDisk/src/main/java/org/bluedb/disk/backup/BackupManager.java

```diff
@@ -194,6 +202,7 @@ private void copyDataFileStraightOver(ReadWriteSegment<?> segment, BlueSerialize
 		}
 	}
 
+	@SuppressWarnings("ResultOfMethodCallIgnored")

```

Comment:
Is IntelliJ telling you to add these? Just curious. Eclipse gives me warnings and says they're not needed.

Reply:
There are a bunch of warnings on methods for things like this in IntelliJ, maybe we need to standardize the warnings settings? I've been trying to keep new code from having any warnings and suppressing these has been needed in cases where we don't want to handle the result.

Reply:
What method is it complaining about? This method returns null... Or is it a method called inside this method? I guess Eclipse is actually saying that `ResultOfMethodCallIgnored` is an unsupported value.

Reply:
It is line 209, the call to `mkdirs()`. That method has been the most common culprit throughout, since it returns a boolean of whether or not it was created.

I could do some testing on this to see if it is worth handling and retrying or throwing an IOException or something, I believe false could be returned in the case of the directories already existing though. I would need to check on that.

Separately, I prefer to put these warning suppression annotations above the method, primarily just to keep the code itself more readable. I'm okay to put it directly above the line to be really explicit if you'd like, but I have felt as a general rule myself that it is worth the bit of ambiguity to keep it separated. Let me know what you think!

Reply:
Huh, I feel like that is a dumb warning. I wouldn't mind adjusting your IDE to not complain about that. There are lots of times that you call a method and it returns something that you don't need to look at. And it is actually causing a warning to appear in Eclipse for me.

I dunno, I don't really care if these stay in there that much. But I definitely don't think it's worth trying to change the code to get the result and do something with it.